functions/Start-DbaAgentJob.ps1

```diff
@@ -109,6 +112,7 @@ function Start-DbaAgentJob {
         [DbaInstanceParameter[]]$SqlInstance,
         [PSCredential]$SqlCredential,
         [string[]]$Job,
+        [string]$StepName,
```

Comment:
Rename to just `Step`

Then we also need an example added for this parameter.

Reply:
Good point. Will add it later.

Reply:
Example added.

Reply:
Example added.